### PR TITLE
Cross ecosystem issuance sample and documentation

### PR DESCRIPTION
**Pull request by chiarapaskulin**

### CODE REVIEW

File: docs/examples/cross-eco-issuance.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+hide:
+    - navigation
+---
+
+# Issue directly and send
```
Comment (janpieterz) [inline docs/examples/cross-eco-issuance.md]:
```suggestion
# Issue and send
```

File: docs/examples/cross-eco-issuance.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+let createCredentialOfferResponse = await trinsicAlice.credential().createCredentialOffer(createCredentialOfferRequest);
+
+console.log("Credential Offer: " + createCredentialOfferResponse.documentJson);
+```
+
+### Create a wallet and accept the credential offer in ecosystem B
+
+Now we create the holder wallet (Bob's) in ecosystem B, authenticate as it and accept the credential offer. After that, we print
```
Comment (janpieterz) [inline docs/examples/cross-eco-issuance.md]:
```suggestion
Now we create the holder wallet (Bob's) in ecosystem B, authenticate and accept the credential offer. After that, we print
```

**Review by janpieterz — APPROVED** [review]

File: docs/examples/cross-eco-issuance.md (hunk 1 above)
Comment (tmarkovski) [inline docs/examples/cross-eco-issuance.md]:
```suggestion
# Cross ecosystem issuance
```

File: docs/examples/cross-eco-issuance.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+Add the SDK to your project
+
+```
+npm i @trinsic/trinsic
+```
+
+Import the required services
+
+```
+import {AuthenticateInitRequest, CreateWalletRequest, SignatureType, TrinsicService} from "@trinsic/trinsic";
```
Comment (tmarkovski) [inline docs/examples/cross-eco-issuance.md]:
```suggestion
import {CreateWalletRequest, TrinsicService} from "@trinsic/trinsic";
```

I don't think these types are used, can be removed

**Pull request merged**